protoc-gen-go/grpc/grpc.go

```diff
@@ -105,12 +109,15 @@ func (g *grpc) Generate(file *generator.FileDescriptor) {
 		return
 	}
 
+	errorPkg = string(g.gen.AddImport(errorPkgPath))
+	codePkg = string(g.gen.AddImport(codePkgPath))
 	contextPkg = string(g.gen.AddImport(contextPkgPath))
 	grpcPkg = string(g.gen.AddImport(grpcPkgPath))
 
 	g.P("// Reference imports to suppress errors if they are not otherwise used.")
 	g.P("var _ ", contextPkg, ".Context")
 	g.P("var _ ", grpcPkg, ".ClientConn")
+	g.P("var errUnimplemented = func(methodName string) error {\n\treturn ", errorPkg, ".Errorf(codes.Unimplemented, \"method %s not implemented\", methodName)\n}")
```

Comment:
This should just be defined as a real function -- although I'm also fine with not including the method name, as before, and making this a `var` of type `error` again (the caller should know what method they called -- it's effectively redundant to include it here).

Reply:
On the other hand, I think that callers would pass the functions to error handlers. So it makes sense that at the point where the error is reported the traceback shows which function being called returned the error. 

I agree with converting it to a real function.